codalab/client/local_bundle_client.py

```diff
@@ -796,7 +801,8 @@ def new_worksheet(self, name, ensure_exists):
             except UsageError:
                 pass
 
-        self.ensure_unused_worksheet_name(name)
+        if not spec_util.is_dashboard(name):
```

Comment:
Why is dashboard exempt?

if someone does 'cl new dashbaord' we don't want to create one if it exists.